DetectorDescription/DDCMS/plugins/DDDefinitions2Objects.cc

```diff
@@ -12,7 +12,7 @@
 #include "XML/Utilities.h"
 #include "FWCore/ParameterSet/interface/FileInPath.h"
 #include "FWCore/Utilities/interface/thread_safety_macros.h"
-#include "DataFormats/Math/interface/GeantUnits.h"
+#include "DataFormats/Math/interface/CMSUnits.h"
```

Comment:
@vargasa @ianna for the use that is done so far of this include this update should be irrelevant, but it brings full consistency with all the rest of the package

Reply:
@fabiocos. It is for consistency sake. The function used here `cms_units::operators::convertRadToDeg` points to ` angle_units::operators::convertRadToDeg`[(here)](https://cmssdt.cern.ch/lxr/source/DataFormats/Math/interface/CMSUnits.h#0022) which is the same endpoint used by `geant_units` [(here)](https://cmssdt.cern.ch/lxr/source/DataFormats/Math/interface/GeantUnits.h#0027).